client/app/controllers/map.js

```diff
@@ -1,6 +1,8 @@
 import Ember from 'ember';
 
 export default Ember.Controller.extend({
+
+  dismissAllStudentsAlert: false,
```

Comment:
Naming nitpick: This sort of sounds like a command/function ("dismiss the all-students alert"), when it's meant to be an indication of whether something has occurred ("the all-students alert has been dismissed"). Something like `studentsAlertDismissed` might be clearer.